### PR TITLE
bump(x11/tigervnc): 1.14.1

### PR DESCRIPTION
**Pull request by twaik**
I will merge this in 72 hours if nobody minds or when it gets 2+ approves.

Works on my device, but I only started `vncserver`, `vncserver -kill` and `vncserver -xstartup xfce4-session` to ensure I did not break existing behaviour.

### CODE REVIEW

File: x11-packages/tigervnc/build.sh [hunk 1]
```diff
@@ -2,13 +2,12 @@ TERMUX_PKG_HOMEPAGE=https://tigervnc.org/
 TERMUX_PKG_DESCRIPTION="Suite of VNC servers. Based on the VNC 4 branch of TightVNC."
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-# No update anymore. v1.11.x requires support of PAM.
-TERMUX_PKG_VERSION=(1.10.1
+TERMUX_PKG_VERSION=(1.14.1
                     21.1.8)
 TERMUX_PKG_REVISION=41
```
Comment (licy183) [inline x11-packages/tigervnc/build.sh:7]:
```suggestion
```

Comment (twaik) [reply x11-packages/tigervnc/build.sh]: 👍

File: x11-packages/tigervnc/build.sh [hunk 2]
```diff
@@ -2,13 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://tigervnc.org/
 TERMUX_PKG_DESCRIPTION="Suite of VNC servers. Based on the VNC 4 branch of TightVNC."
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-# No update anymore. v1.11.x requires support of PAM.
-TERMUX_PKG_VERSION=(1.10.1
+TERMUX_PKG_VERSION=(1.14.1
                     21.1.8)
-TERMUX_PKG_REVISION=41
 TERMUX_PKG_SRCURL=(https://github.com/TigerVNC/tigervnc/archive/v${TERMUX_PKG_VERSION}.tar.gz
```
Comment (truboxl) [inline x11-packages/tigervnc/build.sh:7]:
```suggestion
TERMUX_PKG_SRCURL=(https://github.com/TigerVNC/tigervnc/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

File: x11-packages/tigervnc/build.sh [hunk 3]
```diff
@@ -2,13 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://tigervnc.org/
...
                    https://xorg.freedesktop.org/releases/individual/xserver/xorg-server-${TERMUX_PKG_VERSION[1]}.tar.xz)
-TERMUX_PKG_SHA256=(19fcc80d7d35dd58115262e53cac87d8903180261d94c2a6b0c19224f50b58c4
+TERMUX_PKG_SHA256=(579d0d04eb5b806d240e99a3c756b38936859e6f7db2f4af0d5656cc9a989d7c
                    38aadb735650c8024ee25211c190bf8aad844c5f59632761ab1ef4c4d5aeb152)
 TERMUX_PKG_DEPENDS="libandroid-shmem, libc++, libgnutls, libjpeg-turbo, libpixman, libx11, libxau, libxdamage, libxdmcp, libxext, libxfixes, libxfont2, libxrandr, libxtst, opengl, openssl, perl, xkeyboard-config, xorg-xauth, xorg-xkbcomp, zlib"
```
Comment (truboxl) [inline x11-packages/tigervnc/build.sh:11]:
```suggestion
TERMUX_PKG_DEPENDS="libandroid-shmem, libc++, libgmp, libgnutls, libjpeg-turbo, libnettle, libpixman, libx11, libxau, libxdamage, libxdmcp, libxext, libxfixes, libxfont2, libxrandr, libxshmfence, libxtst, opengl, openssl, perl, xkeyboard-config, xorg-xauth, xorg-xkbcomp, zlib"
```

File: x11-packages/tigervnc/build.sh (hunk 2 above)
Comment (twaik) [reply x11-packages/tigervnc/build.sh:7]: Is there any specific reason to use `refs/tags`?

Comment (truboxl) [reply x11-packages/tigervnc/build.sh:7]: https://github.com/termux/termux-packages/blob/41f130bfed9aab0b89d6565f55af210fdaaba0d5/packages/boxes/build.sh#L28-L30
https://lore.kernel.org/buildroot/87edqhwvd0.fsf@dell.be.48ers.dk/T/

Comment (twaik) [reply x11-packages/tigervnc/build.sh:7]: Oh, probably we should make linter check this.

**Review by truboxl — APPROVED** [review]

**Review by licy183 — APPROVED** [review]

**Pull request merged**